int32 pan
int32 tilt
